packages/wallets/walletconnect/src/adapter.ts

```diff
@@ -48,25 +69,20 @@ export class WalletConnectWalletAdapter extends BaseSignerWalletAdapter {
 
     private _publicKey: PublicKey | null;
     private _connecting: boolean;
-    private _options: ClientOptions;
-    private _params: ClientTypes.ConnectParams;
+    private _options: SignClientTypes.Options;
     private _client: WalletConnectClient | undefined;
-    private _session: SessionTypes.Created | undefined;
+    private _session: SessionTypes.Struct | undefined;
     private _readyState: WalletReadyState =
         typeof window === 'undefined' ? WalletReadyState.Unsupported : WalletReadyState.Loadable;
+    private _network: WalletAdapterNetwork;
 
     constructor(config: WalletConnectWalletAdapterConfig) {
         super();
 
         this._publicKey = null;
         this._connecting = false;
+        this._network = config.network || WalletAdapterNetwork.Devnet;
```

Comment:
`config` is required and `config.network` also required so probably not needed to default?